contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -480,6 +484,8 @@ zfs_key_config_load(pam_handle_t *pamh, zfs_key_config_t *config,
 			config->unmount_and_unload = 0;
 		} else if (strcmp(argv[c], "prop_mountpoint") == 0) {
 			config->homedir = strdup(entry->pw_dir);
+		} else if (strcmp(argv[c], "debug") == 0) {
+			config->debug = 1;
 		}
 	}
 	return (0);
```

Comment:
Not related to your changes but if IIUC this will silently ignore mistyped or otherwise invalid arguments. 
@felixdoerre Wouldn't it be better to give the user a feedback in such a case?

Reply:
Yes, we could print out a warning for unhandled options.